ComponentMissingMessageForm.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace DTALauncherStub;
+
+using System;
+using System.Diagnostics;
+using System.Windows.Forms;
+
+internal sealed partial class ComponentMissingMessageForm : Form
+{
+    public ComponentMissingMessageForm()
+    {
+        InitializeComponent();
+    }
+
+    private void LblDotNetLink_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
+    {
+#pragma warning disable SA1312 // Variable names should begin with lower-case letter
```

Comment:
Yeah I am not a fan of introducing style checks so strict that you have to do stuff like this. This one doesn't even make sense because it's a discard.